tests/tiup-cluster/script/cmd_subtest.sh

```diff
@@ -49,8 +49,9 @@ function cmd_subtest() {
     tiup-cluster $client _test $name writable
 
     # check the data dir of tikv
-    tiup-cluster $client exec $name -N 172.19.0.102 --command "grep /home/tidb/deploy/tikv-20160/data /home/tidb/deploy/tikv-20160/scripts/run_tikv.sh"
-    tiup-cluster $client exec $name -N 172.19.0.103 --command "grep /home/tidb/my_kv_data /home/tidb/deploy/tikv-20160/scripts/run_tikv.sh"
+    # it's ok to omit client type after deploy
+    tiup-cluster exec $name -N 172.19.0.102 --command "grep /home/tidb/deploy/tikv-20160/data /home/tidb/deploy/tikv-20160/scripts/run_tikv.sh"
```

Comment:
Can you add some cases to test the local executor?

Reply:
The local executor has been tested in unit tests. And there is no need to test it in integration testing since it's used as an interface and we have test two of it's implemention (system and builtin), I think.

Too much useless test will take much test resource and make it harder to merge a PR.